share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -186,7 +186,7 @@ spack:
     - amrex +cuda cuda_arch=70
     # - axom +cuda cuda_arch=70 ^umpire@4.1.2 ~shared
     - caliper +cuda cuda_arch=70
-    - chai +cuda ~benchmarks ~tests cuda_arch=70 ^umpire@4.1.2 ~shared
+    - chai +cuda ~benchmarks ~tests cuda_arch=70 ^umpire~shared+cuda
```

Comment:
@davidbeckingsale I think I added that "+cuda", but I am not sure anymore if that changed anything.
Should we try to remove it?